docs/data/stashcache/run-stash-origin-container.md

```diff
@@ -79,6 +84,46 @@ user@host $ docker run --rm --publish 1094:1094 \
 Replacing `<HOST PARTITION>` with the host directory containing data that your origin should serve.
 See [this section](#populating-origin-data) for details.
 
+
+For Kubernetes (with examples):
+
+```file
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: osdftest
+spec:
+  replicas: 1
+    spec:
+      containers:
+        image: opensciencegrid/stash-origin:3.6-release
+        command: [ "sleep" ]
+        args: [ "infinity" ]
```

Comment:
`command` in a deployment refers to the `ENTRYPOINT` so this will result in a container that starts up and does nothing.